src/utils/network/session-helper.js

```diff
@@ -156,12 +154,10 @@ export const getContainerUrlAndFiles = (solidDataset) => {
  */
 
 export const getContainerUrl = (session, fileType, fetchType, otherPodUsername) => {
-  let POD_URL;
-  if (fetchType === 'self-fetch') {
-    POD_URL = String(session.info.webId.split('profile')[0]);
-  } else {
-    POD_URL = `https://${otherPodUsername}.${SOLID_IDENTITY_PROVIDER.split('/')[2]}/`;
-  }
+  const POD_URL =
```

Comment:
Another binary case, this time with fetchType. Replacing if/else with ternary

Reply:
If `fetchType` is binary, why not just pass it in as a boolean?

Reply:
Think we can leave this as is for now.